mcodingbot/utils/peps.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+import dataclasses
```

Comment:
I'd change this to a from import for consistency

Reply:
`@dataclasses.dataclass` is ugly + generally you only use `dataclass` from it so I fully agree